java-test-samples/fargate-dynamodb-sync/spingboot-with-dynamodb/src/test/java/com/amazon/aws/sample/controller/CustomerControllerTest.java

```diff
@@ -0,0 +1,117 @@
+package com.amazon.aws.sample.controller;
+
+import com.amazon.aws.sample.mapper.CustomerMapper;
+import com.amazon.aws.sample.model.Customer;
+import com.amazon.aws.sample.model.CustomerView;
+import com.amazon.aws.sample.service.CustomerServiceImpl;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.util.StreamUtils;
+
+import java.util.Collections;
+import java.util.List;
+
+import static java.nio.charset.Charset.defaultCharset;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@WebMvcTest(CustomerController.class)
+class CustomerControllerTest {
+
+    @Autowired
+    MockMvc mockMvc;
+    @Mock
+    CustomerServiceImpl customerService;
+    @Mock
+    CustomerMapper mapper;
+
+    @InjectMocks
+    CustomerController customerController;
+
+    private static final String customerId = "1";
```

Comment:
Reusing variables across tests is not a best practice. See if we can make them local to the test.

Reply:
Corrected as suggested, committed & pushed the changes.